### PR TITLE
TPP-00008: Blank Calendar Page

### LINKED ISSUE #8: TPP-00008: Blank Calendar Page
> As a user of Menstruation Nation, I want to be able to see a page with a calendar with all my period information.

**Technical Information:**

- Create a new blank page under a subdirectory of the main app.
- Add one blank calendar to the page. 
  - [NOT REQUIRED] It does not currently need to be stylized like the one in the Figma, but if you are feeling adventurous, try to add more than one calendar like in the prototype!

[Figma Link](https://www.figma.com/file/YpKp60yVbVi4fEcQcQRoX3/TPP-User-Testing?node-id=94%3A9338) 

**Good Resources:**

You will need to install the following package. Consult the docs on the github page of the library.

[React Native Calendars Library](https://www.npmjs.com/package/react-native-calendars)

### PR DESCRIPTION
**Pull request by helen-li**
## Description  
- [Ticket Link](https://github.com/uoftblueprint/the-period-purse/issues/8)
- Added blank calendars to the page

### CODE REVIEW

File: tpp-app/src/home/Calendar.js [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
-    return (
-        <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-            <Text>Calendar!</Text>
-        </View>
-    )
+
+export default () => {
+    return (<CalendarList
+        // Callback which gets executed when visible months change in scroll view. Default = undefined
+        onVisibleMonthsChange={(months) => {console.log('now these months are visible', months);}}
```
Comment (yuemily24) [inline tpp-app/src/home/Calendar.js]: Don't think the `console.log` is necessary anymore. Otherwise everything looks good and it works perfectly! :D

**Review by yuemily24 — CHANGES_REQUESTED** [review]
One tiny change, otherwise looks good to me!

**Review by annie951977 — APPROVED** [review]
Very clear with your comments and style, good work

**Review by yuemily24 — APPROVED** [review]
Looks good!

**Pull request merged**